consultation_analyser/consultations/views.py

```diff
@@ -12,6 +12,22 @@ def show_question(request: HttpRequest, consultation_slug: str, section_slug: st
         slug=question_slug, section__slug=section_slug, section__consultation__slug=consultation_slug
     )
     themes_for_question = models.Theme.objects.filter(answer__question=question)
-    # TODO - probably want counts etc.
-    context = {"question": question, "themes": themes_for_question}
+    
+    # Get counts
+    highest_theme_count = 0
```

Comment:
A more Django-y way to get the counts, and the highest count is to use annotations. For example:

```
from django.db.models import Count, Max
themes_for_count = themes_for_question.annotate(answer_count=Count("answer"))
max_count = themes_for_count.aggregate(Max("answer_count"))["answer_count_max"]
```

This should be more efficient (especially when we have more data in there).

More info: https://docs.djangoproject.com/en/5.0/topics/db/aggregation/